src/components/users/users.njk

```diff
@@ -71,36 +71,35 @@
     </tr>
   </thead>
   <tbody class="govuk-table__body">
-    {% for user in users %}
+    {% for guid, user in users %}
     <tr class="govuk-table__row">
       <td class="govuk-table__cell">
         {% if isAdmin or isManager %}
-          <a href="{{ linkTo('admin.organizations.users.edit', {organizationGUID: organization.metadata.guid, userGUID: user.metadata.guid}) }}" class="govuk-link">
-            {{ user.entity.username }}
+          <a href="{{ linkTo('admin.organizations.users.edit', {organizationGUID: organization.metadata.guid, userGUID: guid}) }}" class="govuk-link">
+            {{ user.username }}
```

Comment:
I'm not sure what's the reasoning for shortening/flattening the data is, but we've decided back in the day to do it the long way `user.[entity,metadata].property` in order to keep the consistency with the API as it confused people a little.

I don't feel strongly really, prefer the shorten way. But it's still worth to keep in mind.

Reply:
It was really just to reduce the provided data a little: rather than shoving around big chunky objects, just the elements we know we need are sent to the template.

Believe it's ok, but good to keep in mind.